server/src/main/java/org/candlepin/resource/ConsumerTypeResource.java

```diff
@@ -61,7 +61,8 @@ public ConsumerTypeResource(ConsumerTypeCurator consumerTypeCurator, I18n i18n)
         this.i18n = i18n;
     }
 
-    @ApiOperation(notes = "Retrieves a list of Consumer Types", value = "list")
+    @ApiOperation(notes = "Retrieves a list of Consumer Types", value = "list",
+        response = ConsumerType.class, responseContainer = "list")
```

Comment:
Looks like [`Wrapped`]( https://docs.jboss.org/resteasy/docs/1.0.0.GA/javadocs/org/jboss/resteasy/annotations/providers/jaxb/Wrapped.html) is the same idea but RESTEasy specific.  Swagger didn't pay attention to it because I had to put in the response class and response container to get the client to generate correctly.